### DISCUSSION TITLE
登录有安全问题

### DISCUSSION THREAD

**Discussion by cloudmoonocus** [Binaryify/NeteaseCloudMusicApi#1568] [Q&A]
用了一个月了没问题，半个月前就不行了

**Comment by cloudmoonocus** [comment]
![image](https://user-images.githubusercontent.com/98140702/180604433-204a0e5c-537f-48b1-8c3a-5d612bbf7f29.png)

**Reply by Binaryify** [reply]
用二维码登录

**Reply by cloudmoonocus** [reply]
好的，那得重做了😫😂
